test_scripts/API/VehicleData/GenericNetworkSignalData/commonGenericNetSignalData.lua

```diff
@@ -220,6 +220,22 @@ local function VehicleDataItemsWithDataTableCreation()
     bodyInformationParams.passengerDoorAjar.value = false
     bodyInformationParams.rearLeftDoorAjar.value = false
     bodyInformationParams.rearRightDoorAjar.value = false
```

Comment:
Do we still need support deprecated parameters? (`driverDoorAjar`, `passengerDoorAjar`, `rearLeftDoorAjar`, `rearRightDoorAjar`)

Reply:
Agreed to retain these